gtest-parallel

```diff
@@ -346,6 +358,7 @@ if options.format == 'raw':
   pass
 elif options.format == 'filter':
   logger = FilterFormat()
+  logger.zip_file = os.path.join(options.output_dir, 'passing_test_logs.zip')
```

Comment:
This zip file will be created in interactive runs too—maybe not what we want. How about adding an option, e.g. --save_passing_test_logs=foo.zip, that sets the name of the zip file, and causes no zip file to be created if it isn't set?